### DISCUSSION TITLE
Missing

### DISCUSSION THREAD

**Discussion by amandamayra** [amazonlinux/amazon-linux-2022#341] [Q&A]
Hello, 

I'm working on a project where we use amazon-linux-extras, but after updating to AL2023 we are getting an error that it can't be found. I've seen in some forums that it has been discontinued, could someone inform me if there is any substitute for this package? Or is it necessary to downgrade to Amazon Linux 2?

**Comment by rlove** [comment]
What do you need from amazon-linux-extras?

**Comment by stewartsmith** [comment]
Please see the FAQ for AL2023 https://aws.amazon.com/linux/amazon-linux-2023/faqs/

> Q: Does AL2023 have Amazon-Linux-Extras like AL2?

> A: No, AL2023 does not have extras. For higher-level software packages like language runtimes, we will use the quarterly release where we will add major/minor updates to packages as separate namespaced packages in addition to the default package provided in the repository. For example, default Python version in Amazon Linux 2023 may be 3.8, but we will add Python 3.9 (python39) as a separate namespaced package whenever it is made available. These additional packages will closely follow their upstream release cadence and support model and their support policies can be accessed by the package manager for compliance and security use cases. Default packages will continue to be supported throughout the life of AL2023.

**Reply by ozbenh** [reply]
Note that this is an old example, AL23 went out with Python 3.9 as default and ships 3.11 namespaced
